proposals/2021-05-gravity-bridge-deployment/README.md

```diff
@@ -0,0 +1,174 @@
+
+The oracle was designed to adhere to the Cosmos Hub validator security model without slashing conditions to ensure that a consensus failure on Ethereum does not affect operation of the Cosmos chain.
+
+### Slashing Conditions Spec
+
+https://github.com/cosmos/gravity-bridge/blob/main/spec/slashing-spec.md
+
+## How does it work?
+
+Gravity consists of 4 parts:
+
+- An Ethereum contract called Gravity.sol
+- The Gravity Cosmos module of Gaia
+- The orchestrator program which is run by Cosmos validators alongside the Gravity module
+- A market of relayers, who compete to submit transactions to Ethereum
+
+### Gravity.sol
+
+The Gravity Ethereum contract is a highly compact and efficient representation of weighted powers voting on Ethereum. It contains an Etheruem key from each Cosmos validator, as well as their voting power. This signer set is continuously updated as validation power changes on Cosmos, ensuring that it matches the current Cosmos validator set.
+
+Sending tokens, or updating the validator set, contained in Gravity.sol requires more than 66% of the total voting power to approve the action. In this way Gravity.sol mirrors Tendermint consensus on the Cosmos chain as closely as possible on Ethereum.
+
+### Gravity Cosmos Module
+
+The Gravity module governs and coordinates the bridge. Generating messages for the validators to sign with their Ethereum keys and providing these signatures to relayers who assemble and submit them to the Ethereum chain.
+
+### Orchestrator
+
+The Gravity bridge orchestrator performs all the external tasks the Gravity bridge requires for validators, which includes submission of signatures and submission of Ethereum events.
+
+While the Gravity module concerns itself with the correctness and consensus state of the bridge, the Orchestrator is responsible for locating and creating the correct inputs.
+
+### Market of Relayers
+
+Relayers are an unpermissioned role that observes the Cosmos chain for messages ready to be submitted to Ethereum.
+
+The relayer then packages the validators signatures into an Ethereum transaction and submits that transaction to the Ethereum blockchain. All rewards in the Gravity bridge design are paid to msg.sender on Ethereum. This means that relayers do not require any balance on the Cosmos side and can immediately liquidate their earnings into ETH while continuing to relay newer messages.
+
+## Security assumptions
+
+The Gravity bridge is designed with the assumption that the total amount of funds in Gravity.sol is less than the value of the validator set’s total staked tokens.
+
+If this assumption does not hold true, it would be more profitable for validators to steal the funds in the bridge and simply lose their stake to slashing.
+
+There is no automated enforcement of this assumption. It is up to the $ATOM holders to take action if the amount deposited in the bridge exceeds the total value of all stake on the hub.
```

Comment:
Can we be a bit more precise regarding "taking action" here?

Reply:
Validators could either ensure more tokens enter the staking pool or take action to move what assets they control back across the bridge to reduce the total value in the bridge.

An option to allow the validators to disable deposits might be warranted, but would be complex to implement at this stage.